packages/medusa/src/api/routes/admin/customers/update-customer.ts

```diff
@@ -31,6 +37,16 @@ import { validator } from "../../../../utils/validator"
  *           password:
  *             type: string
  *             description: The Customer's password.
+ *           groups:
+ *             type: Group[]
```

Comment:
This doc-string is used to generate our OpenAPI specs for the reference docs and as such the fields here should be compliant with the [OpenAPI specification](https://swagger.io/specification/). As such we can't use our own TS types such as Group here. In this case we should be using `array`